weaver/wps_restapi/colander_extras.py

```diff
@@ -176,6 +176,12 @@ def __init__(self, *args, v_prefix=False, rc_suffix=True, **kwargs):
 
 
 class ExtendedBoolean(colander.Boolean):
+
+    def __init__(self, *args, true_choices=None, **kwargs):
+        if true_choices is None:
+            true_choices = ("true", "1")
+        super(ExtendedBoolean, self).__init__(true_choices=true_choices, *args, **kwargs)
```

Comment:
Add a changelog entry related to that. 
It is important to indicate that values will not be automatically converted to `bool` anymore. 
They will require explicit `false|true`.

In order to align with OpenAPI definitions
> type: boolean represents two values: true and false. Note that truthy and falsy values such as "true", "", 0 or null are not considered boolean values. 

Let's remove the `"1"` and also override the `false_choices` to be only `"false"`. 

I would like also a comment above those changes that indicate the reason they are defined as `"true"` and `"false"` is because colander converters the value to string lowercase to compare with other thruty/falsy values it should accept. 
The comment should warn not to add other values like `"1"`, etc. to avoid conflict with Integer type for schemas that support both variants.